packages/pwa-kit-runtime/src/utils/logger.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {createLogger, format, transports, config} from 'winston'
+
+const validLogLevels = Object.keys(config.npm.levels)
+
+const logLevel = validLogLevels.includes(process.env.TESTMRT_LOG_LEVEL)
+    ? process.env.TESTMRT_LOG_LEVEL
+    : 'info'
```

Comment:
I guess this means in MRT we'd want to set the `TESTMRT_LOG_LEVEL` env var to `warn` by default for Production envs.